### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Level6*

## Solution

- *[Fix linter warning](https://github.com/YunoHost-Apps/monitorix_ynh/commit/bcc9e992e1b5886b0cd86845a2a19162c23c35f1)*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -65,9 +65,7 @@ ynh_script_progression --message="Installing dependencies..." --weight=7
 ynh_install_app_dependencies $pkg_dependencies
 tempdir="$(mktemp -d)"
 ynh_setup_source --dest_dir="$tempdir"
-ynh_package_update
-dpkg --force-confdef --force-confold -i /$tempdir/app.deb
-ynh_package_install -f
```
Comment (Josue-T) [inline scripts/install:70]: Well why did you remove this line and the ynh_package_update ?
And why did you remove all dpkg parameters that I putt ?

**Review by Josue-T — CHANGES_REQUESTED** [review]
